jnigen/test/simple_package_test/c_based/dart_bindings/simple_package.dart

```diff
@@ -359,6 +456,109 @@ class Example extends jni.JObject {
         .fromRef(_getRandomNumericString(reference, random.reference).object);
   }
 
+  static final _protectedMethod = jniLookup<
+          ffi.NativeFunction<
+              jni.JniResult Function(
+                  ffi.Pointer<ffi.Void>,
+                  ffi.Pointer<ffi.Void>,
+                  ffi.Pointer<ffi.Void>)>>("Example__protectedMethod")
+      .asFunction<
+          jni.JniResult Function(ffi.Pointer<ffi.Void>, ffi.Pointer<ffi.Void>,
+              ffi.Pointer<ffi.Void>)>();
+
+  /// from: protected void protectedMethod(java.lang.String a, java.lang.String b)
+  void protectedMethod(
```

Comment:
Do we want protected stuff to be generated?

Reply:
https://github.com/dart-lang/jnigen/pull/116#discussion_r1008168946